client/src/components/FlashCard/CreateFlashcard.js

```diff
@@ -0,0 +1,96 @@
+import React, {useState} from "react"
+import { Button, Stack, TextField } from "@mui/material"
+import axios from 'axios'
+
+const CreateFlashcard = ({ userId, deckId }) => {
+  // how can we use state here to make sure we're validating info
```

Comment:
Even if it is code you're inheriting, you should not leave in unnecessary comments or files.